src/keysignature.js

```diff
@@ -108,7 +108,6 @@ Vex.Flow.KeySignature = (function() {
       var glyph_width = glyph_data.width + extra_width;
       this.width += glyph_width;
       // Set the width and place the glyph on the stave
-      glyph.setWidth(glyph_width);
```

Comment:
Given that `glyph_width` here includes `extra_width`, do you know how it affects the final rendering of key signatures?